internal/pkg/service/buffer/config/worker.go

```diff
@@ -42,6 +42,7 @@ type WorkerConfig struct {
 	RetryFailedFiles        bool             `mapstructure:"enable-retry-failed-files" usage:"Enable retry for failed files."`
 	TasksCleanup            bool             `mapstructure:"tasks-cleanup-enabled" usage:"Enable periodical tasks cleanup functionality."`
 	TasksCleanupInterval    time.Duration    `mapstructure:"tasks-cleanup-interval" usage:"How often will old tasks be deleted."`
+	ReaderPageSize          int              `mapstructure:"reader-page-size" usage:"Page size of etcd range queries."`
```

Comment:
Vramci zvysovania pripusnosti je dobre mat tuto hodnotu konfigurovatelnu.
Je to velkost stranky, ked za z etcd nacitavaju data na upload do S3.
Mala hodnota znamena vela requestov, velka hodnota zase vacsie vyuzitie memory.